modules/mauth-test-utils/src/main/java/com/mdsol/mauth/test/utils/ProtocolTestSuiteHelper.java

```diff
@@ -32,7 +32,7 @@
       byte[] jsonData = Files.readAllBytes(normalizePath(configFile));
       tmp = objectMapper.readValue(jsonData, SigningConfig.class);
     } catch (IOException ex) {
-      ex.printStackTrace();
+      throw new IllegalStateException("Failed to load the config file " + configFile);
```

Comment:
Please pass `ex` through as the `cause`.

```
hrow new IllegalStateException("Failed to load the config file " + configFile, ex)
```

Also, please apply this fix to everywhere else where you're catching IOException.

Reply:
updated as suggested in https://github.com/mdsol/mauth-jvm-clients/pull/133/commits/5fa7a0869d5fdd0f4a11d1af1be669f59672df18